config/wikis/trwiki.yaml

```diff
@@ -30,17 +32,24 @@ autolabeled_samples:
 merged_samples:
   labeled_revisions.20k_2015:
     - human_labeled_revisions.20k_2015
+  labeled_revisions.3k_2020:
+    - human_labeled_revisions.3k_2020
+
+concatenated_samples:
+  labeled_revisions.23k_2015_2020:
+    - labeled_revisions.20k_2015
+    - labeled_revisions.3k_2020
```

Comment:
This will change to `labeled_revisions.20k_2020` and the large sample will be something like "40k_2015_2020"